trade-finance-manager-ui/server/controllers/utilisation-reports/edit-payment/index.ts

```diff
@@ -47,13 +45,15 @@ export const getEditPayment = async (req: GetEditPaymentRequest, res: Response)
         redirectTab,
         errors,
       );
-      return renderEditPaymentPage(res, editPaymentViewModel);
+
+      return res.render('utilisation-reports/edit-payment.njk', editPaymentViewModel);
     }
 
-    const editPaymentViewModel = getEditPaymentViewModel(paymentDetails, reportId, paymentId, isCheckboxChecked, redirectTab, errors);
-    return renderEditPaymentPage(res, editPaymentViewModel);
+    const editPaymentViewModel: EditPaymentViewModel = getEditPaymentViewModel(paymentDetails, reportId, paymentId, isCheckboxChecked, redirectTab, errors);
+
+    return res.render('utilisation-reports/edit-payment.njk', editPaymentViewModel);
```

Comment:
Could we extract the template path into a const and use that on line 49, line 99 and this line to avoid duplication please

Reply:
done